docs/examples/modal/mobileExample.js

```diff
@@ -0,0 +1,40 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Flex, Layer, Modal, Box, Text, DeviceTypeProvider, Button } from 'gestalt';
+
+export default function Example(): Node {
+  const [showModal, setShowModal] = React.useState(false);
```

Comment:
nit: import `useState` directly (here and the other examples)

(I'm writing a lint rule for this, just not finished yet 😅)

Reply:
DONE